### PR TITLE
Reorg OWNERS and introduced OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by rhuss**
This update is for aligning with the other Knative projects so
that a group like the TOC list can be updated with less efforts
in the future.

Also I took the occasion to clean up a bit, so removed
@cppforlife and @sixolet as they are not active in the project since
at least one year. Happy to re-add both accounts again in the future.

Thanks a lot Naomi and Dmitriy for all your contribution and kicking off the client !

### CODE REVIEW

**Comment by rhuss** [comment]
@knative/technical-oversight-committee I reorganized the OWNERS files here but also removed inactive (over 1 year) approvers without actually knowing if this is the right process. If this is not ok, please let us know and point us to the correct way for keeping the OWNERS file healthy.

/hold

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

Inactive folks can speak up, we can always add them back imo.

**Comment by mattmoor** [comment]
If Naomi isn't active, then we should probably update https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#client as well.  I'll add this to the agenda for Thursday.

**Comment by rhuss** [comment]
> If Naomi isn't active, then we should probably update https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#client as well. I'll add this to the agenda for Thursday.

I already added a PR https://github.com/knative/community/pull/456 and would hold this issue until this has been eventually be merged.

Going to add this issue to the TOC project ...

**Review by maximilien — APPROVED** [review]
/ok-to-test
/lgtm

**Pull request merged**
